common/cont-lib/usr/share/cont-docs/70-general.txt

```diff
@@ -0,0 +1,16 @@
+General container help
+----------------------
+
+Run `docker run THIS_IMAGE container-usage` to get this help.
```

Comment:
@mfojtik wrote:
> It is possible to have two version of this help?  One for platform/language
> images, where this helper is not needed and second for database images where
> you have to override the default CMD to get the help?
> This is not a strong requirement, we will most likely be doing it in
> OpenShift by layering this image with STI scripts and poiting the CMD to
> usage script.

If you do `docker run THIS_IMAGE` on platform/language images, you'll get
the 'container-usage' automatically.  Do I understand your concerns right?

Those commands are equivalent on "platform/language" images:
  ```
  docker run THIS_IMAGE
  docker run THIS_IMAGE container-usage
  ```
But to me, the help should say the explicit version..

Pavel